docs/editors/new-editor.md

```diff
@@ -78,7 +78,7 @@ example, in vim-lsc the `window/logMessage` notification is always displayed in
 the UI so `-Dmetals.status-bar=log-message` can be configured to direct
 higher-priority messages to the logs. However, whenever possible, if the client
 supports the ability to add in `experimental` items to the `ClientCapabilities`
-interface, this is preferable.
+interface and/or to `InitializationOptions`, this is preferable.
```

Comment:
```suggestion
interface or to `InitializationOptions`, this is preferable.
```
I'd say just use **or**. We don't want confusion over whether they should set them both or not.

Reply:
I was trying to say that some setting should be part of `InitializationOptions` other in `experimental`

Reply:
Understood 👍